app/src/main/java/com/example/concordia_campus_guide/Models/Coordinates.java

```diff
@@ -45,4 +47,6 @@ public double getLongitude() {
     public void setLongitude(double longitude) {
         this.longitude = longitude;
     }
+
+    public LatLng getLatLng() { return new LatLng(this.longitude, this.latitude); }
```

Comment:
You are returning longitude, latitude in the LatLng object, please make sure this is the desired behavior, and if it is, I think we should dissect this and find root cause because it would mean that we are using the LatLng object incorrectly

![image](https://user-images.githubusercontent.com/35609369/77783968-20d82a00-7030-11ea-84e9-06cb9c102d11.png)